### ISSUE TITLE
Should we require 2 reviews to merge a PR?

### ISSUE THREAD

**Issue by fitzthum** [confidential-containers/guest-components#341]
As discussed in the community meeting on 7/31, it could be valuable to unify our contribution requirements across subprojects. This might incentivize us to require 2 reviews on each PR, but does that actually make sense? Do we have enough reviewers for each of the guest components? My hunch is that we might not and that requiring two reviewers could lower velocity. wdyt?

@Xynnn007 @jialez0 @jiangliu @sameo

**Comment by Xynnn007** [comment]
Yes. I think the aim to have more reviewers is to avoid destructive or buggy PR. I agree that there are not enough reviewers who have both time and are familiar with the code in guest.

 Another choice is that each PR should be approved by at least two persons from different companies. However 
will this be somehow formalism?

**Comment by mythi** [comment]
> This might incentivize us to require 2 reviews on each PR, but does that actually make sense?

Having (at least) two code owners (in `CODEOWNERS`) per subsystem/area and requiring one such approval would probably be better.

**Comment by sameo** [comment]
> > This might incentivize us to require 2 reviews on each PR, but does that actually make sense?
> 
> Having (at least) two code owners (in `CODEOWNERS`) per subsystem/area and requiring one such approval would probably be better.

I second that. I don't think there's a magic number of reviews per PR, it depends on the numbers of maintainers/owners for a given area. So it may make sense to have 2 for e.g. the operator but 1 for smaller subsystems.
So we should at least enforce having a `CODEOWNERS` file for all projects, one that covers all the code base for each project. And then decide on a case by case if we require 1 or 2 approvals.
